gr-dtv/lib/dvbt2/dvbt2_paprtr_cc_impl.h

```diff
@@ -94,6 +94,11 @@ class dvbt2_paprtr_cc_impl : public dvbt2_paprtr_cc
                          float vclip,
                          int iterations,
                          unsigned int vlength);
+
+    // Disable copy because raw pointers.
+    dvbt2_paprtr_cc_impl(const dvbt2_paprtr_cc_impl&) = delete;
```

Comment:
Which members are raw pointers? Do you refer to `p2_papr_map_*k`?

Reply:
`p2_papr_map` and `tr_papr_map`.

But hmm… actually those point to const static lifetime objects, so are safe to copy. Removing these lines.